### PR TITLE
frontend: added prop verticallyCentered for View component

### PR DESCRIPTION
**Pull request by shonsirsha**
To fulfil our UI requirements, View needs to be able to centre its content vertically when NOT on full screen (`!fullscreen`).

### CODE REVIEW

File: frontends/web/src/components/view/view.tsx [hunk 1]
```diff
@@ -41,6 +42,7 @@ type TViewProps = {
  * @param minHeight optional minimum height, useful for keeping content area same size through multiple views
  * @param onClose if a callback is provided it will render a close button that triggers the callback
  * @param textCenter centers all text content in the view
+ * @param verticallyCentered centers all text content in the view
```
Comment (thisconnect) [inline frontends/web/src/components/view/view.tsx:45]: verticallyCentered make sense

the docstring text implies it should also work also for fullscreen, which would actually be nice.

I made a POC with a few other changes: https://github.com/shonsirsha/bitbox-wallet-app/pull/1  wdyt?

Comment (shonsirsha) [reply frontends/web/src/components/view/view.tsx:45]: > the docstring text implies it should also work also for fullscreen, which would actually be nice.

I don't think I can see any difference when `fullScreen` and `verticallyCentered` are both `true` as opposed just having `fullScreen` to be true (setting fullScreen automatically vertically centres it..?)

**Comment by thisconnect** [comment]
afaik this doesn't create a merge conflict (I tried merging into staigng-pocket and then merged staging-pocket into master).

@shonsirsha should I do final testing then we merge this, or do you want to merge this into staging-pocket?

**Comment by shonsirsha** [comment]
@thisconnect No hard preference, but I think it's fine to merge to master after you're done with the final testing..

**Review by thisconnect — APPROVED** [review]
tested LGTM

File: frontends/web/src/routes/device/bitbox02/bitbox02.tsx [hunk 2]
```diff
@@ -455,13 +456,14 @@ class BitBox02 extends Component<Props, State> {
...
+            verticallyCentered
             withBottomBar
-            width="600px">
+            width="690px">
             <ViewHeader title={t('button.unlock')}>
               <p>{t('bitbox02Wizard.stepConnected.unlock')}</p>
             </ViewHeader>
             <ViewContent fullWidth>
-              {attestationResult === false ? (
+              {attestationResult === true ? (
```
Comment (Beerosagos) [inline frontends/web/src/routes/device/bitbox02/bitbox02.tsx:466]: Was this put into the commit by mistake? Looks like a regression to me cc @shonsirsha @thisconnect

Comment (shonsirsha) [reply frontends/web/src/routes/device/bitbox02/bitbox02.tsx:466]: Yes by looking at the code this isn't right. Or at least this PR shouldn't concern anything about `attestationResult`. I'll make a PR to revert this bit. good catch!

Comment (shonsirsha) [reply frontends/web/src/routes/device/bitbox02/bitbox02.tsx:466]: Possibly it was added when [this](https://github.com/shonsirsha/bitbox-wallet-app/pull/1) was created and this attestationResult part was only meant for testing during development. Feel free to clarify this though @thisconnect :)

**Pull request merged**